AdminRouter/config/adminrouter.conf

```diff
@@ -0,0 +1,576 @@
+#
+# This is the main Apache HTTP server configuration file.  It contains the
```

Comment:
Can we delete all the comments ? I assume they are coming from the sample config file, but without them it's easier to read the config template.

Reply:
It's updated with comments removed. It's indeed more easy to read. Thanks for this good suggestion